packages/devcmd-cli/src/index.ts

```diff
@@ -73,6 +73,19 @@ async function startProcess(command: string, args: Array<string>, dirPath: strin
 async function runInDevCmdsDir(dirPath: string): Promise<void> {
   const [, , ...args] = process.argv;
 
+  if (args.length > 0) {
+    const cmdName = args[0].trim();
+
+    if (cmdName.length > 2 && cmdName.indexOf("--") === 0) {
+      try {
+        await startProcess("node", ["-e", `require('devcmd/from-cli').runReserved('${cmdName.slice(2)}')`], dirPath);
```

Comment:
note that this will fail when `cmdName.slice(2)` contains a single quote (e.g. running `devcmd --hel'p`), with an error message that's probably not easy to understand for a user

Reply:
This was merged with the normal runInDevCmdsDir function so this comment is probably obsolete now